data/card_catalogue/social.json

```diff
@@ -44,7 +56,10 @@
     "upgrade": "",
     "tier": "blue",
     "details": {
-      "rules": "Speak with any living NPC. Return this card to the deck and draw another.",
+      "rules": {
+        "active": "Speak with any living NPC, then draw another blue card.",
+        "passive": ""
+      },
```

Comment:
changed Commune and Seance to this wording so that the player can keep the card if they like it (at least until their hand is full). makes it more appropriate for character creation too.

Reply:
🤔 There's some thinking to be done about card pickups. If I can then pick seance back up into my hand at the end of the session and keep this additional card I drew, then this generates blue cards. I think I'll start up an issue to concretize how exactly returning cards to your hand at the end of the session works because we haven't been doing it in practice.

Reply:
That issue aside, I think we need to buff these cards to the point where we don't need to add the draw clause.

My heuristic for blues is that they should work pretty consistently as a distraction. I think we can achieve that here by emphasizing the spooky aspect of this effect. If we made it clear that commune was speaking inside their head, it could easily be an effective distraction even if the NPC is right in front of you. For seance, if everyone around could hear the dead speaking to you, that'd cause a distraction.

Reply:
> 🤔 There's some thinking to be done about card pickups.

Second this thought. These are my assumptions at the moment, will open an issue for this at some point:

**Hand**: 15 card max, your hand in tts. The only place you can cast from under normal circumstances.

**Inventory**: *aka library, pool, something better than inventoryh* unlimited size, shared between players, physically just the non-map area of the table.

**Drawing Cards**: Cards are drawn into the hand until it overflows, then they're added to the player inventory.

**Casting**:
1. Player plays card from hand
2. We deal with effect
3. We return the card to the group/player inventory.

This card in particular could be balanced with a "draw from inventory" clause. I don't think this is a good solution for this card in lieu of what you suggested in your other comment, just wanted to illutrate an idea for getting a card back that doesn't generate them. The one good thing about that idea is that it makes the players think of using it in conjuction with another right away which would make for some fun situations esp. with seance.

Reply:
Dedicated an issue to this:

- #125